pkg/target/regolib/src.rego

```diff
@@ -320,10 +324,35 @@ matches_namespaces(match) {
 
 matches_namespaces(match) {
   has_field(match, "namespaces")
+  # JULIAN - I can't figure out why we care about namespace selectors here.  Is that because nobody uses namespaceSelectors alongside namespaces?
```

Comment:
This function is confusingly named "ns_selectors" means all namespace selectors, not just label selectors. Basically, we don't apply namespace selection criteria for cluster scoped resources unless the resource is itself a Namespace.